src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -9,59 +9,56 @@
 // except according to those terms.
 
 // ignore-cross-compile
-// ignore-pretty
 
 #![feature(quote, rustc_private)]
 
 extern crate syntax;
 
-use syntax::ast;
-use syntax::codemap;
-use syntax::parse;
-use syntax::print::pprust;
+use syntax::codemap::DUMMY_SP;
+use syntax::print::pprust::*;
 
-trait FakeExtCtxt {
```

Comment:
I'm testing that not only (e.g.) `quote_expr!(cx, 23)` works and produces 23, but also that `quote_expr!(cx, $expr)` (where `expr` is the result of the earlier quote) still produces 23.